back-up-and-restore-data.md

```diff
@@ -105,11 +106,14 @@ INSERT INTO episodes (id, season, num, title, stardate) VALUES
 ### Dump just a table's schema
 
 ~~~ shell
-$ cockroach dump startrek episodes --user=maxroach --dump-mode=schema > backup.sql
+$ cockroach dump startrek episodes --insecure --user=maxroach --dump-mode=schema > backup.sql
 ~~~
 
 ~~~ shell
 $ cat backup.sql
+~~~
```

Comment:
Why are you splitting the command and its output into multiple boxes? Isn't that misleading?